src/main/java/filemanager/serviceImpl/ConverterFromJsonToXmlServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package filemanager.serviceImpl;
+
+import com.google.inject.Inject;
+import filemanager.service.ConverterFromJsonToXmlService;
+import filemanager.service.JsonReader;
+import filemanager.service.JsonToXmlConverter;
+import filemanager.service.XmlWriter;
+import org.json.JSONObject;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class ConverterFromJsonToXmlServiceImpl implements ConverterFromJsonToXmlService {
+
+    @Inject
+    private JsonReader reader;
+
+    @Inject
+    private JsonToXmlConverter converter;
+
+    @Inject
+    private XmlWriter writer;
```

Comment:
That is common practice to use ```@inject``` annotation once on constructor and assign provided values to class properties. That also makes testing easier. Can you please follow that practice?